examples/batch/CMakeLists.txt

```diff
@@ -0,0 +1 @@
+
```

Comment:
I saw the cmake support was taken off in the simple example too. Should I add that to both?

Reply:
Yes, let's have CMake support for both.